components/object-table/src/test.js

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import { mount } from 'enzyme';
+
+import Component from '.';
+
+describe('ObjectTable', () => {
+  let fields = [
```

Comment:
same comments WRT how the `ArrayObjectTable` test file is structured apply here too

Reply:
@stevesims I think I will raise an issue to look at this test structuring in another branch.